src/analog.jl

```diff
@@ -149,11 +149,11 @@ end
 function analog_output(t::AOTask, channel::String; range=nothing)
     if range == nothing
         device::String = split(channel,'/')[1]
-        range=float(analog_output_ranges(device)[end,:])
+        range=float(analog_voltage_output_ranges(device)[end,:])
```

Comment:
good catch!  next time though, please put this in a separate PR.  fixing a bug like this doesn't have anything to do with updating to the latest version of the drivers.